README.md

```diff
@@ -100,7 +109,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is a file that contains the objects of a database. We represent a one-to-many relationship within the class of the objects in volved using  "belongs_to" has "has_many"
```

Comment:
Try using the word "structure" in your answer. Databases themselves don't have objects per se.